src/smart-components/group/add-group/users-list-itless.js

```diff
@@ -167,8 +167,8 @@ const UsersListItless = ({ selectedUsers, setSelectedUsers, userLinks, usesMetaI
   };
 
   const toolbarDropdowns = () => {
-    const onToggle = (isOpen) => {
-      setIsToolbarDropdownOpen(isOpen);
+    const onToggle = () => {
+      setIsToolbarDropdownOpen(!isToolbarDropdownOpen);
```

Comment:
It might be better to use sync function on this one
```suggestion
      setIsToolbarDropdownOpen(isToolbarDropdownOpen => !isToolbarDropdownOpen);
```